src/yafowil/common.py

```diff
@@ -333,7 +363,7 @@ def display_proxy_renderer(widget, data):
                 input_attrs = input_attributes_full(widget, data, value=val)
                 rendered += data.tag('input', **input_attrs)
         else:
-            rendered += input_generic_renderer(widget, data)
+            rendered = input_generic_renderer(widget, data, pos='after')
```

Comment:
See above, if position really needs to be set here hardcoded (for what exactly?) it should be done here, otherwise revert please